disposebag/src/test/kotlin/com/kizitonwose/android/disposebag/DisposeBagTest.kt

```diff
@@ -19,11 +19,14 @@ class DisposeBagTest {
     private lateinit var lifecycleOwner: TestLifecycleOwner
     private lateinit var testScheduler: TestScheduler
 
+    private lateinit var disposeBag: DisposeBag
```

Comment:
This `disposeBag` is used in only one test case, I really don't see the need for a field reference.

Reply:
The reason why I did this is just to explicitly have the class under test as a field. I guess it makes it clearer the object we are testing.